pai-management/Dockerfile

```diff
@@ -52,20 +53,33 @@ RUN apt-get -y update && \
     git clone https://github.com/Microsoft/pai.git &&\
     pip install python-etcd docker kubernetes
 
+WORKDIR /tmp
+
 ENV JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
 
+RUN echo "source /usr/share/bash-completion/completions/git" >> ~/.bashrc
+
 # Only node manager need this.#
-#COPY docker-17.06.2-ce.tgz /usr/local
 RUN wget https://download.docker.com/linux/static/stable/x86_64/docker-17.06.2-ce.tgz
-RUN cp docker-17.06.2-ce.tgz /usr/local
-RUN tar xzvf /usr/local/docker-17.06.2-ce.tgz
-RUN cp -r /docker/* /usr/bin/
+RUN tar xzvf docker-17.06.2-ce.tgz
+RUN mv docker/* /usr/local/bin/
+
+# alert manager tool
+RUN wget https://github.com/prometheus/alertmanager/releases/download/v0.15.2/alertmanager-0.15.2.linux-amd64.tar.gz
+RUN tar xzvf alertmanager-0.15.2.linux-amd64.tar.gz
```

Comment:
Alertmanager is launched at dev-box container? Why should make it at this container

Reply:
this is not for launch alert manager in dev-box, it's just for download amtool which is a client that can mute alert.

Reply:
ok